Makefile

```diff
@@ -943,14 +943,14 @@ endif
 test-go-bench: PACKAGES = $(shell grep --exclude-dir api --include "*_test.go" -lr testing.B .  | xargs dirname | xargs go list | sort -u)
 test-go-bench: BENCHMARK_SKIP_PATTERN = "^BenchmarkRoot"
 test-go-bench: | $(TEST_LOG_DIR)
-	go test -run ^$$ -bench . -skip $(BENCHMARK_SKIP_PATTERN) -benchtime 1x $(PACKAGES) \
+	go test -run ^$$ -bench . -skip $(BENCHMARK_SKIP_PATTERN) -count=6 -benchtime 1x $(PACKAGES) \
```

Comment:
Do we take advantage of the increased runs in any way? I can see this being part of a larger change that makes use of the bench results, but on its own I'm not sure it adds much. (Even so I'm a bit skeptical that every benchmark we have was built with this in mind.)

Also notice that we are still running 1x, so even though the counts are higher the benchmark itself is still minimal.

Reply:
Should we increase the benchtime too?

Reply:
I think it's hard to pitch a generic number, my 2c is that the flags should be on a per-benchmark basis. It also has some to do with how long we want the jobs to take.

Reply:
Should probably just remove `benchtime` since it defaults to 1s. With `-count=6` and no `benchtime` each benchmark would take a minimum of 6s which may balloon the total time. I can test it later to see how long it takes. The current workflow run for the current change took about 10mins though.

We probably will want flags on a per-benchmark basis eventually since not all benchmarks are made equal. For now I think gathering some data with this change would give more insight into which benchmarks may need special care.

Reply:
Yeah it seems we have too many benchmarks that adding 6s for each one really balloons it. I think this is a good first step at least to get some data.

Reply:
It sounds like we'll want to revert to -benchtime=1x but I'll leave my stamp regardless. Thanks for the all the explanations here, folks.

Reply:
Oh, one last suggestion: what about '-benchtime=6x' instead of '-benchtime=1x -count=6`?